src/NuGet.Core/NuGet.PackageManagement/BuildIntegration/PackageReferenceRollbackException.cs

```diff
@@ -13,6 +13,7 @@ namespace NuGet.PackageManagement
     /// and additional log messages with specific information on
     /// what caused the rollback.
     /// </summary>
+    [SerializableAttribute]
     public class PackageReferenceRollbackException : InvalidOperationException
```

Comment:
This class needs to implement `ISerializable` properly for `LogMessages` to round trip.